test/versioned/elastic/package.json

```diff
@@ -11,18 +11,20 @@
         "node": ">=16"
       },
       "dependencies": {
-        "@elastic/elasticsearch": "7.13.0"
+        "@elastic/elasticsearch": "7.13.0",
+        "@elastic/transport": "8.4.1"
```

Comment:
this isn't needed.  this is only testing on 7.13.0 which doesn't have `@elastic/transport`